var/spack/repos/builtin/packages/py-deeptools/package.py

```diff
@@ -14,6 +14,7 @@ class PyDeeptools(PythonPackage):
     # The test suite and associated test data is missing in the pypi tarball.
     url      = "https://github.com/deeptools/deepTools/archive/3.3.0.tar.gz"
```

Comment:
Now that we're no longer running the test suite, it would be better to download from PyPI using the new `pypi` attribute: https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#pypi

This will also require changing the checksums.